mne_bids/mne_bids.py

```diff
@@ -582,6 +597,24 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     if verbose:
         print('Writing data files to %s' % raw_file_bids)
 
+    # check to see if there is a headspace (hsp) or electrode placement
+    # (electrode) provided.
+    # if so, copy it to the correct location
+    hsp_ext = (_parse_ext(hsp)[1] if hsp is not None else None)
+    electrode_ext = (_parse_ext(electrode)[1] if electrode is not None
+                     else None)
+    exts_same = hsp_ext == electrode_ext
+    # what to use as standard names?
+    d = {'HEADPOINTS': hsp, 'ELECTRODES': electrode}
```

Comment:
I have put some default values here for the acquisition parameters in the case that the file types are the same (as they are for the test KIT data). There is however no actual specification on what the acquisition names *should* be so we can probably set them as whatever want.